docs/development/setup.md

```diff
@@ -0,0 +1,156 @@
+# Local Development Setup
+
+- [Prerequisites](#prerequisites)
+- [Preperation](#preperation)
+- [Run libvirt-provider for local development](#run-libvirt-provider-for-local-development)
+- [Interact with the `libvirt-provider`](#interact-with-the-libvirt-provider)
+- [Deploy `libvirt-provider`](#deploy-libvirt-provider)
+
+> ℹ️ **NOTE**:</br>
+> To be able to take exec console of the machine, you can follow either of the below approaches:</br>
+> - Run the `libvirt-provider` as the `libvirt-qemu` user.</br>
+> - Create an entry with `devpts /dev/pts devpts rw,nosuid,noexec,relatime,gid=5,mode=0660 0 0` in `/etc/fstab`.</br>
```

Comment:
is it needed, if libvirt-provider runs as libvirt-qemu user?

Reply:
As stated in line 10, you can choose either of the approaches listed. I have included all the options here because we have not yet reached a consensus on which one to follow.

Reply:
you have right, my fault